rest/config_legacy.go

```diff
@@ -117,12 +117,13 @@ func (lc *LegacyServerConfig) ToStartupConfig() (*StartupConfig, DbConfigMap, er
 			continue
 		}
 		bsc = &BootstrapConfig{
-			Server:       *dbConfig.Server,
-			Username:     dbConfig.Username,
-			Password:     dbConfig.Password,
-			CACertPath:   dbConfig.CACertPath,
-			X509CertPath: dbConfig.CertPath,
-			X509KeyPath:  dbConfig.KeyPath,
+			Server:                         *dbConfig.Server,
+			Username:                       dbConfig.Username,
+			Password:                       dbConfig.Password,
+			CACertPath:                     dbConfig.CACertPath,
+			X509CertPath:                   dbConfig.CertPath,
+			X509KeyPath:                    dbConfig.KeyPath,
+			AllowInsecureServerConnections: base.BoolPtr(true),
```

Comment:
Still need this to be false by default in the default case. Think this should mean just removing this as the current DefaultStartupConfig() should handle it